scss/modifierClasses/_display.scss

```diff
@@ -14,6 +14,7 @@ Class                   | Description
 `.display--inline`      | sets `display` to `inline`
 `.display--inlineBlock` | sets `display` to `inline-block`
 `.display--inlineFlex`  | sets `display` to `inline-flex`
+`.display--grid`        | sets `display` to `grid`
```

Comment:
I'm not sure `display--grid` has much use as a modifier, but `at[Breakpoint]_display--grid` would be useful for things that are grid-displayed that are displayed differently at smaller breakpoints.

Reply:
We get those automatically by adding `grid` to the `$display-values` list